### PR TITLE
Blog post for the Java guide preview

### PR DESCRIPTION
**Pull request by jponge**

### CODE REVIEW

**Comment by cescoffier** [comment]
Good for me, @vietj when do you want it published?

**Comment by jponge** [comment]
I'd say publish ASAP so we can gather as much community feedback and publish along with the release.

**Comment by vietj** [comment]
I think now is fine, we'll twit it today and monday from the twitter account

**Comment by vietj** [comment]
when it's published it is not set in stone and can get updated as well I think

**Comment by jponge** [comment]
Ping?

File: src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Preview of a guide for Java developers
+template: post.html
+date: 2017-06-09
+author: jponge
+draft: true
+---
+
+I could not attend the last Vert.x community face-to-face meeting last fall, but one item that was discussed is the need for guides aimed at certain types of developers.
```
Comment (vietj) [inline src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md]: should be Eclipse Vert.x as it is the first occurence

File: src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## We need your feedback!
+
+The code is available at [https://github.com/vert-x3/vertx-guide-for-java-devs](https://github.com/vert-x3/vertx-guide-for-java-devs).
+You can report feedback as Github issues and even offer pull-requests.
+
+You can check it out from GitHub (the AsciiDoc is being rendered fine from the repository interface) or you can check out pre-rendered HTML and PDF versions that I am temporarily sharing and keeping up-to-date from my Dropbox: [https://www.dropbox.com/sh/ni9znfkzlkl3q12/AABn-OCi1CZfgbTzOU0jYQpJa?dl=0](https://www.dropbox.com/sh/ni9znfkzlkl3q12/AABn-OCi1CZfgbTzOU0jYQpJa?dl=0)
+
+Many thanks to Thomas Segismont and Julien Viet who contributed some parts, and also to the people who reviewed it privately.
+
+As usual, we welcome your feedback!
```
Comment (vietj) [inline src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md]: should say how we collect feedback, e.g vertx-dev or an issue in the repo

Comment (jponge) [inline src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md]: It's explained a few lines above:

> You can report feedback as Github issues and even offer pull-requests.

**Comment by jponge** [comment]
Uo

File: src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md (hunk 2 above)
Comment (vietj) [inline src/site/blog/posts/2017-06-09-preview-of-guide-for-java-developers.md]: right

**Comment by vietj** [comment]
published

**Pull request merged**